# origin: Horms <horms@verge.net.au>
# inclusion: submitted upstream
# descrition: [PATCH] earlyquirk.o is needed for CONFIG_ACPI_BOOT
#
#   [PATCH] earlyquirk.o is needed for CONFIG_ACPI_BOOT
#   
#   Signed-off-by: Horms <horms@verge.net.au>
#
===== arch/i386/kernel/earlyquirk.c 1.1 vs 1.2 =====
--- a/arch/i386/kernel/Makefile	2005-08-01 16:26:02.000000000 +0900
+++ b/arch/i386/kernel/Makefile	2005-08-01 16:26:20.000000000 +0900
@@ -36,11 +36,11 @@
 obj-$(CONFIG_X86_CPUID)		+= cpuid.o
 obj-$(CONFIG_MICROCODE)		+= microcode.o
 obj-$(CONFIG_APM)		+= apm.o
+obj-$(CONFIG_ACPI_BOOT)		+= acpi.o earlyquirk.o
-obj-$(CONFIG_ACPI_BOOT)		+= acpi.o
 obj-$(CONFIG_ACPI_SLEEP)	+= acpi_wakeup.o
 obj-$(CONFIG_SMP)		+= smp.o smpboot.o trampoline.o
 obj-$(CONFIG_X86_LOCAL_APIC)	+= mpparse.o apic.o nmi.o
+obj-$(CONFIG_X86_IO_APIC)	+= io_apic.o
-obj-$(CONFIG_X86_IO_APIC)	+= io_apic.o earlyquirk.o
 obj-$(CONFIG_X86_VISWS_APIC)	+= visws_apic.o
 obj-$(CONFIG_CPU_EMU486)	+= emu.o
 obj-$(CONFIG_EDD)             	+= edd.o
